lib/components/PlaylistPage.js

```diff
@@ -1,15 +1,78 @@
 import React from 'react';
+import Request from 'superagent';
+
 import SearchBar from './SearchBar';
+import SearchResult from './SearchResult';
+import SearchResults from './SearchResults';
 
 const INPUT_KLASS = "spotify-search"
 const INPUT_PLACEHOLDER = "Search Spotify"
 
 export default React.createClass({
+  getInitialState() {
+    return {
+      playlistTracks : {},
+      searchResults : [],
+    }
+  },
+
+  handleSearch(e) {
+    e.preventDefault();
+    let queryValue = e.target.value;
+    this.searchSpotify(queryValue);
+  },
+
+  storeResults(tracks) {
+    debugger;
+    this.state.searchResults = tracks;
+    this.setState({ searchResults : self.state.searchResults });
+  },
```

Comment:
@damwhit commented on this earlier but I still think there could be some issues here:

you are attempting to set state in line 27 `this.state.searchResults = tracks;`
- This should not set state given you did not call `this.setState`
- Then in line 28 you attempt to take your newly set state and properly set state with it (I think this is where things go wrong)

One solution could be to delete line 27 & 28 and just set state once:
`this.setState({ searchResults : tracks });`